src/epd2in13_v2/command.rs

```diff
@@ -128,18 +128,60 @@ pub (crate) enum DeepSleepMode {
     MODE_2 = 0x11,
 }
 
-pub struct GateDrivingVoltage(pub i32);
-pub struct SourceDrivingVoltage(pub i32);
+pub struct GateDrivingVoltage(pub u8);
+pub struct SourceDrivingVoltage(pub u8);
+pub struct VCOM(pub u8);
 
 pub trait I32Ext {
+    fn vcom(self) -> VCOM ;
     fn gate_driving_decivolt(self) -> GateDrivingVoltage;
     fn source_driving_decivolt(self) -> SourceDrivingVoltage;
 }
 
 impl I32Ext for i32 {
+
+    // This is really not very nice. Until I find something better, this will be
+    // a placeholder.
+    fn vcom(self) -> VCOM {
+        assert!(self >= -30 && self <= -2);
+        let u = match -self {
+            2 => 0x08,
+            3 => 0x0B,
+            4 => 0x10,
+            5 => 0x14,
+            6 => 0x17,
+            7 => 0x1B,
+            8 => 0x20,
+            9 => 0x24,
+            1 => 0x28,
```

Comment:
is this really 1 and not 10 :thinking: would be a strange order

Reply:
You're right, thanks !